tests/test_utils/logging_setup.py

```diff
@@ -35,7 +36,8 @@ def emit(self, record):
 def setup_logging():
     """Configure the python logging appropriately for the tests.
 
-    (Logs will end up in _trial_temp.)
+    Logs will end up in _trial_temp. Exceptions are additionally
```

Comment:
Not everything logged at `logger.error` is an exception! `s/exceptions/errors/`.

Likewise in the changelog, and the name of the handler.